applications/nrf_desktop/doc/settings_loader.rst

```diff
@@ -33,7 +33,14 @@ For details on the default configuration alignment, see the following sections.
 Settings backend
 ================
 
-By default, nRF Desktop devices use non-volatile storage settings backend (:kconfig:option:`CONFIG_SETTINGS_NVS`).
+By default, the nRF Desktop application, depending on the non-volatile memory technology used by the device, uses one of the following settings backend:
+
+* :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` - see the following :kconfig:option:`CONFIG_SETTINGS_ZMS` Kconfig option for more details.
+* :ref:`Non-Volatile Storage (NVS) <zephyr:nvs_api>` - see the following :kconfig:option:`CONFIG_SETTINGS_NVS` Kconfig option for more details.
+
+ZMS is used for the devices with the RRAM or MRAM non-volatile memory that do not require explicit erase.
+Otherwise, the NVS is used.
```

Comment:
I would simplify it a bit:

```suggestion
* :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` (:kconfig:option:`CONFIG_SETTINGS_ZMS`) - used for the devices with non-volatile memory that does not require explicit erase (MRAM, RRAM).
* :ref:`Non-Volatile Storage (NVS) <zephyr:nvs_api>` (:kconfig:option:`CONFIG_SETTINGS_NVS`) - used for the devices with non-volatile memory that requires explicit erase (FLASH).
```

We should point to doc page instead of Kconfig option help for details